app/bettermail.go

```diff
@@ -24,33 +26,62 @@ func init() {
 	http.HandleFunc("/hook-test-harness", hookTestHarnessHandler)
 	http.HandleFunc("/test-mail-send", testMailSendHandler)
 	http.HandleFunc("/_ah/bounce", bounceHandler)
+	http.HandleFunc("/test-subject", testSubjectHandler)
+}
+
+type EmailThread struct {
+	CommitSHA string
```

Comment:
Should use `datastore:",noindex"` to avoid App Engine indexing these properties (which adds extra overhead).